Task3_Position_the_Cubes/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <div class="Parent">1

```

Comment:
The text content '1' should be inside the closing tag of the parent div, like this: <div class='Parent'>1</div>